### PR TITLE
Update docker files

### PR DESCRIPTION
**Pull request by kulikovee**
https://github.com/kulikovee/skilljs/issues/4

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
-RUN echo 'APT::Install-Recommends "0";' >> /etc/apt/apt.conf.d/00-docker
-RUN DEBIAN_FRONTEND=noninteractive \
-  apt-get update \
-  && apt-get install -y python3 \
-  && rm -rf /var/lib/apt/lists/*
-RUN useradd -ms /bin/bash apprunner
-USER apprunner
+FROM node:20.3.1-bullseye
+WORKDIR /src
+COPY package.json package-lock.json ./
```
Comment (arkhipenkavitali) [inline Dockerfile]: COPY package*.json ./ 
 ;)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      - "3000:3000"
+    stdin_open: true
+    tty: true
+    volumes:
+      - ./
+    networks:
+      - default
+
+volumes:
+  mysql-volume:
```
Comment (arkhipenkavitali) [inline docker-compose.yml]: problem is here, need an absolute path to this lib
but i couldnt solve the problem

File: Dockerfile (hunk 1 above)
Comment (kulikovee) [reply Dockerfile]: Fixed, thanks

File: docker-compose.yml (hunk 2 above)
Comment (kulikovee) [reply docker-compose.yml]: Removed, but I faced with a new issue ([docker-log.txt](https://github.com/kulikovee/skilljs/files/11994654/docker-log.txt)):

`ERROR: for nextjs-docker-container  Cannot start service nextjs-docker-container: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: open /dev/ptmx: no such file or directory: unknown`

image

Comment (kulikovee) [reply docker-compose.yml]: Fixed all the problems, ready to review!

**Review by arkhipenkavitali — APPROVED** [review]

**Pull request merged**